performance/jsonParsers/jsonify/jsonify_json_parser.js

```diff
@@ -0,0 +1,7 @@
+
+var x;
+(function (exports) {
+  exports.parse_json_jsonify = function parse_json_jsonify(input) {
+    x = jsonifyParse(input);
```

Comment:
I actually don't know what the `var x` stuff is for. When I made the first new one I copied from one of the other parsers and it had the `var x`. I left it in there in case I discovered it was needed somehow. Later I saw it wasn't and didn't put it in the other ones. However, I didn't remove it from the *jsonify* one, woops.